Podfile.lock

```diff
@@ -45,15 +45,15 @@ PODS:
   - Braintree/Venmo (4.17.0):
     - Braintree/Core
     - Braintree/PayPalDataCollector
-  - BraintreeDropIn (6.4.1):
-    - BraintreeDropIn/DropIn (= 6.4.1)
-  - BraintreeDropIn/DropIn (6.4.1):
+  - BraintreeDropIn (6.5.0):
```

Comment:
Why are these changing?

Reply:
`6.5.0` is the latest version of iOS Drop-In

Reply:
Yes, but why are these changing in this PR and not before? Was that a mistake before?

Reply:
Ah, sorry for not clarifying. It isn't necessarily a mistake, but the `Podfile.lock` file always lags the current release because it's self-referencing. This PR contains a new version because cocoapods now has the latest version.